### PR TITLE
chore(#8185) Remove Guided Tour and Welcome message

### PR DESCRIPTION
**Pull request by latin-panda**
# Description

- Removes the welcome message modal
- Removes the guided tour modal
- Removed related code of guided tour in e2e tests 

medic/cht-core#8185

[CHT Docs PR](https://github.com/medic/cht-docs/pull/1026)

Note: **Do not merge - this is for 4.3.0**

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs

If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8185-remove-guided-tour/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8185-remove-guided-tour/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8185-remove-guided-tour/docker-compose/cht-couchdb-clustered.yml)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: webapp/tests/karma/ts/services/startup-modals.service.spec.ts
Comment (latin-panda) [inline webapp/tests/karma/ts/services/startup-modals.service.spec.ts:1]: This file was removed in [this PR](https://github.com/medic/cht-core/pull/8194)

File: webapp/src/ts/services/startup-modals.service.ts
Comment (latin-panda) [inline webapp/src/ts/services/startup-modals.service.ts:1]: This file was removed in [this PR](https://github.com/medic/cht-core/pull/8194)

File: webapp/src/js/bootstrap-tour-standalone.js [hunk 1]
```diff
@@ -1,1703 +0,0 @@
...
-/* ========================================================================
- * Bootstrap: transition.js v3.3.7
- * http://getbootstrap.com/javascript/#transitions
- * ========================================================================
- * Copyright 2011-2016 Twitter, Inc.
- * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
- * ======================================================================== */
-
-
-+function ($) {
```
Comment (dianabarsan) [inline webapp/src/js/bootstrap-tour-standalone.js:31]: Removing this makes me very happy!

File: webapp/src/ts/services/startup-modals.service.ts [hunk 2]
```diff
@@ -20,19 +15,9 @@ interface StartupModal {
...
-    // welcome screen
-    {
-      required: settings => !settings.setup_complete,
-      render: () => {
-        return this.modalService.show(WelcomeComponent, { class: 'welcome' })
-          .catch(() => {});
-      },
-    },
-    // guided setup
+    // Guided Setup
```
Comment (dianabarsan) [inline webapp/src/ts/services/startup-modals.service.ts:20]: I suppose this file has to go after both tour and guided setup are removed, right?

**Review by dianabarsan — APPROVED** [review]
Kill it with fire!

File: webapp/src/ts/services/startup-modals.service.ts (hunk 2 above)
Comment (latin-panda) [reply webapp/src/ts/services/startup-modals.service.ts:20]: Correct!

**Pull request merged**